### DISCUSSION TITLE
does anyone have any alternatives to flash a usb/sd card on an enrolled chromebook that cant install extensions?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1128] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
does anyone have any alternatives to flash a usb/sd card on an enrolled chromebook that cant install extensions?

**Comment by AshtonDavies** [comment, marked as answer]
Google does not provide any alternatives to the two options. They are very cheap to purchase, and anyone who is somehow limited on money spending or access to an external memory device should not be performing such things to begin with.
